source/cfp.rst

```diff
@@ -0,0 +1,107 @@
+Call For Proposals
+==================
+
+We are planning to use Google Forms for handling CFP proposals.
+
+Daniele has set up a form with the following questions:
+
+About you
+---------
+
+We'd like to know a little bit about you for our files. If you submit more than
+one proposal, you don't need to send all of these details again.
```

Comment:
Ah, I misread, but I think it could be clearer.

Could we make it clear which fields we don't need again?  As far as I can tell, this is just "More information about you (public)" and "More information about you (private)".  Can we add "Leave blank if you've filled these fields in on another submission"?